libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/utils/RouteUtils.java

```diff
@@ -65,19 +70,73 @@ public boolean isLastLeg(RouteProgress routeProgress) {
    */
   @Nullable
   public List<Point> calculateRemainingWaypoints(RouteProgress routeProgress) {
```

Comment:
wouldn't it be more user friendly if we'd return empty list? wouldn't need to handle null down the road.